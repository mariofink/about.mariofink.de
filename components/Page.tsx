interface props {
  children: React.ReactNode;
}

const Page = ({ children }: props) => {
  return <div className="container mx-auto">{children}</div>;
};

export default Page;
