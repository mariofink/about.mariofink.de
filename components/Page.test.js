import React from "react";
import renderer from "react-test-renderer";
import Page from "./Page";

describe("Page", () => {
  it("renders", () => {
    const tree = renderer.create(<Page>My Page</Page>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
