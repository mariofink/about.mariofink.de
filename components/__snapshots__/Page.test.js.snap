// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<div
  className="container mx-auto"
>
  My Page
</div>
`;
